src/buffer.rs

```diff
@@ -233,13 +224,14 @@ impl AudioBuffer {
         AudioBuffer::from_channels(channels, sample_rate)
     }
 
-    /// Resample to the desired sample rate.
```

Comment:
Please leave the `///` in place. If you run rustdoc with `--document-private-items`, it's nice to have this available.
Also put the example usage in three slashes. Not sure why this was not the case already.

Reply:
done (there are ignored from `cargo test --doc` though)